devops/scripts/deploy-public-upstream.sh

```diff
@@ -75,6 +92,18 @@ echo "Copying README to docroot."
 rm ./README.md
 cp /tmp/composer-managed-README.md ./README.md
 
+# Create a list of files that we need to exclude from the commit.
+# These are files that may be coming from Roots or old commits that do not exist in the source and _should not exist_ on the upstream.
+ignored_files="composer.lock CODE_OF_CONDUCT.md CONTRIBUTING.md"
+
+# Remove ignored files from the commit.
+for file in $ignored_files; do
```

Comment:
this should be an array not a space delineated string but I care 2/10

Reply:
I thought space separated string _was_ how you did arrays in bash. Historically, we ran into issues creating arrays (which probably means we were __doing_it_wrong) and this is how I remember doing it so it worked.

Reply:
this works, its just not ideal